### ISSUE TITLE
An option to use the default new page format when adding a new page while editing a PDF file

### ISSUE THREAD

**Issue by Photon89** [xournalpp/xournalpp#3087]
**Is your feature request related to a problem? Please describe.**
I use Xournal++ for video lessons, in particular, for fill in the blanks worksheets. When the worksheet is done, I would like to start an empty new page to do some exercises. Currently, when I press the "new page" button, it duplicates the last (more precisely, last active) page of of the PDF file, that is, takes the last PDF file's page as the background of the new page.

**Describe the solution you'd like**
I would like the new page to have the default background (squared withborders in my case) when it is created, even if I'm in the context of annotating a PDF.

**Describe alternatives you've considered**
Possibly this should be optional, as there might be use cases where duplicating the last PDF page makes sense (though I cannot think of any right now).

**Comment by rolandlo** [comment]
This should be easily possible via a [plugin](https://xournalpp.github.io/guide/plugins/plugins/). Basically you need
```Lua
app.uiAction({["action"]="ACTION_NEW_PAGE_AFTER"})
app.changeCurrentPageBackground("graph")
```

**Comment by Technius** [comment]
This can be done by unchecking `Copy current page background instead` in `Journal > Configure new page template`.

**Comment by Photon89** [comment]
@Technius Thanks, works! This default new page settings menu is really well hidden... :smiley:

**Comment by Technius** [comment]
> This default new page settings menu is really well hidden.

We can probably improve discoverability by adding a configuration button to the new page popup.

**Comment by Esokrates** [comment]
> @Technius Thanks, works! This default new page settings menu is really well hidden... smiley

Yes, that's why I think why this should be moved to the `Edit -> Preferences -> Defaults` settings tab: https://github.com/xournalpp/xournalpp/issues/2191
Neither me nor many other users discovered that menu until we filed issues about it.

**Comment by jayant-tripathi** [comment]
> e those two issues should be me

Its a lovely tool.
I concur with you. Kindly move configure page template to preferences.
